src/runtime_src/core/tools/common/XBHelpMenusCore.cpp

```diff
@@ -442,7 +468,10 @@ XBUtilities::report_subcommand_help( const std::string &_executableName,
 
 
   // -- Options
-  report_option_help("OPTIONS", _optionDescription, _positionalDescription, false, removeLongOptDashes);
+  report_option_help("OPTIONS", _optionDescription, _positionalDescription, false, removeLongOptDashes);
+
+  // -- Custom Section
+  std::cout << customHelpSection << std::endl;

```

Comment:
I think we are not supposed to use std::endl, not sure on this one. Seems like we veer towards using '\n' instead